src/assets/images/illustrations/undraw_mobile_feed_73bd.svg

```diff
@@ -0,0 +1 @@
13.80782.132-18.15284-4.80115-5.2788-4.35917-10.82529-1.47008-16.40375,7.38788-14.265-3.1969-29.44375-13.88428-42.0647a23.66937,23.66937,0,0,0-19.75537-8.29179l-19.7975,1.41411a23.70939,23.70939,0,0,0-21.05293,16.94614v0c-4.72724,6.42917-7.25736,12.84055-5.66438,19.21854-7.08065,4.83882-8.27029,10.67977-5.08851,17.2644,2.698,4.14592,2.66928,8.18161-.12275,12.1056a55.89106,55.89106,0,0,0-8.31012,16.5061Z" transform="translate(-308.72978 -86.11098)" fill="#2f2e41"/><path d="M328.37,741.00271l17.65033,7.22059L430.2606,637.50757l26.4755,129.97064,17.65033-3.20915c11.32243-112.05194,20.555-226.34385,9.22631-280.39962l-65.38646-2.808-29.68465,94.67-14.44118,64.183Z" transform="translate(-308.72978 -86.11098)" fill="#2f2e41"/><path d="M461.54207,813.60773h0a12.67189,12.67189,0,0,0,7.85522-.83484l8.57872-3.86042c7.19328-.36934,12.09123,1.06343,16.09631-4.163a10.49624,10.49624,0,0,0,1.22538-10.5816h0a10.56885,10.56885,0,0,0-8.63109-6.28021l-11.783-7.42484-.61847,5.56623c-5.38534,3.05094-11.77909,1.733-18.5541-1.23694l-3.69822,12.94376A12.67188,12.67188,0,0,0,461.54207,813.60773Z" transform="translate(-308.72978 -86.11098)" fill="#2f2e41"/><path d="M314.86865,787.05638h0a12.67184,12.67184,0,0,0,7.70062,1.76121l9.3674-.86547c6.92339,1.98653,11.09065,4.93215,16.57584,1.28955a10.49625,10.49625,0,0,0,4.59511-9.61025h0a10.56886,10.56886,0,0,0-6.124-8.74262l-8.73335-10.84871-2.39246,5.06375c-6.08422,1.13683-11.70352-2.18588-17.14694-7.19493l-7.701,11.0414A12.67187,12.67187,0,0,0,314.86865,787.05638Z" transform="translate(-308.72978 -86.11098)" fill="#2f2e41"/><circle cx="133.56513" cy="252.14322" r="28.88236" fill="#ffb8b8"/><polygon points="98.264 313.117 135.972 305.094 140.786 273.805 114.31 268.189 98.264 313.117" fill="#ffb8b8"/><path d="M417.424,485.07288h67.39218l-15.19038-38.13769c1.10383-13.12273-4.45793-22.77459-12.41823-31.17762l-10.09892-25.35466c-16.31743-10.73928-28.61911-9.937-41.719,4.01144C389.858,416.405,400.60885,449.46221,417.424,485.07288Z" transform="translate(-308.72978 -86.11098)" fill="#e85b3f"/><path d="M447.91093,566.104l1.91657,21.0823c.68408,7.52478,4.63564,12.72072,12.00148,11.0371h0a12.48084,12.48084,0,0,0,8.82808-16.7496l-6.7004-16.97436-19.25491-78.62422,3.20915-56.96244H415.81942l4.01144,60.97388Z" transform="translate(-308.72978 -86.11098)" fill="#ffb8b8"/><path d="M397.30666,376.55688c3.10544-28.13634,11.06231-52.09549,24.86965-71.017l2.06713-.06949v-.51117l6.71182.28563,18.66347-.62728c-.437.45447-.8538.91109-1.2761,1.36713l13.60834.57909,12.83661,26.4755a85.26577,85.26577,0,0,0-37.99453-2.15737,9.08687,9.08687,0,0,0-7.1006,5.65771,3.38035,3.38035,0,0,0,.02372,2.97414c5.39469,9.40963,5.12585,19.74357-.357,30.59361-1.19687,2.36847,1.10132,5.21956,1.8216,7.83291Z" transform="translate(-308.72978 -86.11098)" fill="#2f2e41"/><path d="M413.41255,430.5173H450.3178l-2.58722-24.07477a16.70084,16.70084,0,0,0-15.85294-15.99889h0a16.70082,16.70082,0,0,0-17.522,16.02556Z" transform="translate(-308.72978 -86.11098)" fill="#e85b3f"/></svg>
```

Comment:
Does you use this svg?

Reply:
No but if we want to add more later I thought those were good ones to use should i remove them?